modules/fedimint-mint-client/src/lib.rs

```diff
@@ -154,6 +155,44 @@ impl OOBNotes {
             .expect("Invariant violated: OOBNotes does not contain any notes")
     }
 
+    pub fn notes_json(&self) -> Result<serde_json::Value, serde_json::Error> {
+        let mut map = serde_json::Map::new();
```

Comment:
Nit: `map` is a meh name for a variable - it doesn't describe the meaning of it in it.